min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
@unho @nijel I'm all for bumping this to prevent the `if<3.5` logic.  But I'd like to know more ito how this impacts users.  Are we actually excluding some platforms now?